### ISSUE TITLE
Question: any chance to get this working with FlexScrollView

### ISSUE THREAD

**Issue by ganySA** [Wenape/fview-infinitescrollview#2]
FlexScrollView is such a great view layout control...

Any chance for a version that works with it? 

https://github.com/IjzerenHein/famous-flex
https://github.com/gadicc/meteor-famous-views

**Comment by Wenape** [comment]
I agree it's a great layout control, I use it now instead of the regular Scrollview. The thing is, this plugin would depend on https://github.com/gadicc/fview-flex/ which depends on https://github.com/IjzerenHein/famous-flex and everything changes to often. Gadi's version for Meteor works great, it just needs a few fixes. Maybe then I can implement a plugin.

For now this is what I do: `FlexScrollviewContainer` would be you ContainerView in which the FlexScrollview is the first child. The `offset` inside the last `if` let's you control when to trigger the callback.

    Template.FlexScrollviewContainer.rendered = function () {
       var fview = FView.from(this);
       var FlexScroller = fview.children[0];
       var sync = new GenericSync([type], { direction : GenericSync.DIRECTION_Y });
       fview.view.pipe(sync);

       sync.on('update', function (data) {
         var displacement = Scroller.view.getAbsolutePosition();

         // Inifinite Scroll
        if (displacement > 0) {
          var offset = (Scroller.view._dataSource[0].view._size[1] * (Scroller.view._dataSource.length - 1)) - displacement - Scroller.view._contextSizeCache[1];
          if (offset < 0 && ! Session.equals('isLoading', true))
            getMoreResults();
          }
       });
    };

**Comment by ganySA** [comment]
Sorry that flew right over my head. Im still pretty new to famo + meteor (3 days new!) so currently i have:

    {{#FlexLayoutController layout='HeaderFooterLayout' layoutOptions=layoutOption}}
         {{#Surface style="z-index:9999" target="header" size="[undefined, 65]"}}
              {{> yield 'header'}}
         {{/Surface}}    
         {{#RenderController transition="slideWindow" target="content"}}
              {{>yield}}
        {{/RenderController}}
        {{#ContainerSurface style="z-index:999999999" target="footer" size="[undefined, 55]"}}
              {{> yield 'footer'}}
         {{/ContainerSurface}}
    {{/FlexLayoutController}}

--- below is a yield template
    {{#ContainerSurface overflow="hidden"}}
        {{#FlexScrollView alwaysLayout="true" overflow="hidden"}}
            {{#famousEach listings}}
                {{>Surface template="mflick" size="[undefined,true]"}}
            {{/famousEach}}
        {{/FlexScrollView}}
    {{/ContainerSurface}}

Would i create a template at the ContainerSurface level?

**Comment by Wenape** [comment]
That's right, so then you can access the `rendered` callback.
